src/Pdf.php

```diff
@@ -218,6 +222,27 @@ public function getImageData($pathToImage)
     }
 
     /**
+     * Return raw image data.
+     * @param string $pathToImage
+     *
+     * @return \Imagick
+     */
+    public function convertRemoteFile($pathToImage)
```

Comment:
We don't want users to call this directly. Make this protected.

There also no conversion being done here. Name this `getRemoteImageData`, reflecting the function named where it is being called.